lib/dynamic_widget/basic/visiblity_widget_parser.dart

```diff
@@ -8,16 +8,15 @@ class VisibilityWidgetParser extends NewWidgetParser {
   @override
   Widget parse(Map<String, dynamic> map, BuildContext buildContext,
       EventListener? listener) {
-    String? visibleString = map['visible'] as String;
-    bool visible = visibleString.parseBool() ?? true;
+    bool? visible = map['visible'];
```

Comment:
is it tested ??, In export their is no change but while parsing now we are considering it as bool instead of string

Reply:
not tested, but looked json supports bool